textstat/textstat.py

```diff
@@ -26,6 +26,13 @@ def legacy_round(number, points=0):
     p = 10 ** points
     return float(math.floor((number * p) + math.copysign(0.5, number))) / p
 
+ordinal_map = {1: 'th', 2: 'nd', 3: 'rd'}
```

Comment:
I think you've got a typo here :blush: 

```suggestion
ordinal_map = {1: 'st', 2: 'nd', 3: 'rd'}
```

Reply:
Good catch, doh